swig/python/setup.py.in

```diff
@@ -38,6 +38,6 @@ setup(
   '@PYTHON_MODULE_NAME@':['$<TARGET_FILE_NAME:@PYTHON_SWIG_TARGET_NAME@>', @FLAMEGPU_CODEGEN_INCLUDE_CLEAN@@FLAMEGPU_INCLUDE_CLEAN@@FLAMEGPU_PYTHON_PACKAGE_DATA_OS_SPECIFIC@],
   },
   install_requires=[
-    'astpretty',
```

Comment:
pathlib is core python since 3.4, it will always be available. 

Additioanlyl, if we're not using pathlib in the pyflamegpu module iteself, its not needed in setup.py, but rather would be a dependency of the individual example (e.g. its not require to use pyflaemgpu).

Reply:
removed